src/components/form/style.css

```diff
@@ -22,35 +24,41 @@ smoothly-form > form > fieldset {
 	border-color: unset;
 	border-image: unset;
 }
-smoothly-form > form > fieldset > * {
+
+smoothly-form>form>fieldset>* {
 	flex-grow: 1;
 	min-width: 10em;
 	flex-basis: 40%;
 }
-smoothly-form > form > fieldset smoothly-picker smoothly-picker-menu ,
-smoothly-form[looks="border"] > form > fieldset smoothly-picker smoothly-input,
-smoothly-form[looks="grid"] > form > fieldset smoothly-picker smoothly-input,
-smoothly-form[looks="border"] > form > fieldset > *,
-smoothly-form[looks="grid"] > form > fieldset > * {
+
+smoothly-form>form>fieldset smoothly-picker smoothly-picker-menu,
+smoothly-form[looks="border"]>form>fieldset smoothly-picker smoothly-input smoothly-input-file,
+smoothly-form[looks="grid"]>form>fieldset smoothly-picker smoothly-input smoothly-input-file,
+smoothly-form[looks="border"]>form>fieldset>*,
+smoothly-form[looks="grid"]>form>fieldset>* {
 	border: rgb(var(--smoothly-default-contrast)) solid 1px;
 }
-smoothly-form[looks="line"] > form > fieldset smoothly-picker smoothly-input,
-smoothly-form[looks="line"] > form > fieldset > * {
+
+smoothly-form[looks="line"]>form>fieldset smoothly-picker smoothly-input smoothly-input-file,
+smoothly-form[looks="line"]>form>fieldset>* {
 	border-bottom: rgb(var(--smoothly-default-contrast)) solid 1px;
 }
-smoothly-form[looks="grid"] > form > fieldset {
+
+smoothly-form[looks="grid"]>form>fieldset {
 	border: rgba(var(--smoothly-default-contrast), .5) solid .5px;
 }
-smoothly-form[looks="grid"] > form > fieldset smoothly-picker smoothly-input,
-smoothly-form[looks="grid"] > form > fieldset > * {
+
+smoothly-form[looks="grid"]>form>fieldset smoothly-picker smoothly-input smoothly-input-file,
```

Comment:
changing the original selector from what it was to this breaks the picker search box